{"WARN": 11, "filename": "5_2_do_not_capture_all_exceptions_warning_.py", "line": 10, "message": "No exception type(s) specified."}
